hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestUtils.java

```diff
@@ -307,31 +299,26 @@ public static String createNewLogFile(FileSystem fs, String basePath, String par
     return fileID;
   }
 
-  public static void createCompactionCommitFiles(FileSystem fs, String basePath, String... instantTimes)
-      throws IOException {
-    for (String instantTime : instantTimes) {
-      boolean createFile = fs.createNewFile(new Path(basePath + "/" + HoodieTableMetaClient.METAFOLDER_NAME + "/"
-          + HoodieTimeline.makeCommitFileName(instantTime)));
-      if (!createFile) {
-        throw new IOException("cannot create commit file for commit " + instantTime);
-      }
-    }
-  }
-
   public static void createCompactionRequest(HoodieTableMetaClient metaClient, String instant,
-      List<Pair<String, FileSlice>> fileSliceList) throws IOException {
+                                             List<Pair<String, FileSlice>> fileSliceList) throws IOException {
```

Comment:
Does this indentation style break the rule of the project?

Reply:
Checkstyle did not complain about (or cover) it. The intellij of the new version starts to format this way. Guess it is due to missing a checkstyle rule for this, causing IDE format in its own way. Right now I've seen this style exist too in the codebase.

Reply:
I see the issue is that we did not set `forceStrictCondition` to `true`, that's why intellij can align this and make the indentation longer than what the rule sets.
https://checkstyle.sourceforge.io/config_misc.html#Indentation_Properties

@yanghua Do you agree we should set it to `true`? This can be done in a separate PR.

Reply:
Let @vinothchandar chime in. I knew that @vinothchandar 's major concern is that whether the rule of code style support automatically formatting or not.

Here I will ignore these issues.